ckan/logic/validators.py

```diff
@@ -454,7 +454,7 @@ def tag_name_validator(value, context):
 
     tagname_match = re.compile('[\w \-.]*$', re.UNICODE)
     if not tagname_match.match(value):
-        raise Invalid(_('Tag "%s" must be alphanumeric '
+        raise Invalid(_('Tag "%s" must consist of alphanumeric '
                         'characters or symbols: -_.') % (value))
```

Comment:
@Chealer do you like this message?

> Tag "%s" can only contain alphanumeric characters, spaces (" "), hyphens ("-"), underscores ("_") or dots (".")

```suggestion
        raise Invalid(_('Tag "%s" can only contain alphanumeric '
                        'characters, spaces (" "), hyphens ("-"), '
                        'underscores ("_") or dots (".")') % (value))
                  
                        
```

Reply:
Thank you @amercader and Gauravp-NEC, I like it a lot, but:

- English is not my native language. I am not fully qualified to confirm the phrasing. English **might** have a more precise/correct name for the "." character than "dot".
- As for the meaning, I doubt that underscores are supported. Your 2019-12-16 comment implied they were, but my reading of current [master's tag_name_validator()](https://github.com/ckan/ckan/blob/master/ckan/logic/validators.py#L455) is that they are not. That being said, Python regular expressions are definitely not my native language neither.